### PR TITLE
Moved WatchManager.js from WatchKeeper

### PR DESCRIPTION
**Pull request by nathanholzworth**
Moved WatchManager.js from WatchKeeper to Kubernetes-utils to make it available to other razee projects and updated it slightly to reflect its new location.

### CODE REVIEW

**Review by adamkingit — CHANGES_REQUESTED** [review]
You'll need to add the new function to the exports to use it outside the kubernetes-util package

File: lib/WatchManager.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2019 IBM Corp. All Rights Reserved.
```
Comment (adamkingit) [inline lib/WatchManager.js:2]:
```suggestion
 * Copyright 2019, 2022 IBM Corp. All Rights Reserved.
```

**Review by adamkingit — CHANGES_REQUESTED** [review]
Suggest changing `let` to `const` wherever possible, and replacing all uses of `var` with `let` or `const` as appropriate

File: lib/WatchManager.js (hunk 1 above)
Comment (nathanholzworth) [reply lib/WatchManager.js]: Should I do this for the index.js too?

**Review by adamkingit — APPROVED** [review]

**Pull request merged**